### PR TITLE
Add backplane-2.0 prow jobs for CLC

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
/hold
https://github.com/stolostron/backlog/issues/19647

### CODE REVIEW

**Comment by elgnay** [comment]
I think we need to update the fast forword setting for each repo as well.
https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/provider-credential-controller/stolostron-provider-credential-controller-main.yaml#L51-L56

File: ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-backplane-2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-curator-controller
+    env:
+      IMAGE_REPO: cluster-curator-controller
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_COMPONENT_NAME=cluster-curator-controller
```
Comment (hanqiuzh) [inline ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-backplane-2.0.yaml:50]: please specify the repo & prefix (e.g.
https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/managed-serviceaccount/stolostron-managed-serviceaccount-backplane-2.0.yaml#L86-L87 )

File: ci-operator/config/stolostron/clusterclaims-controller/stolostron-clusterclaims-controller-backplane-2.0.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: clusterclaims-controller
+    env:
+      IMAGE_REPO: clusterclaims-controller
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
```
Comment (hanqiuzh) [inline ci-operator/config/stolostron/clusterclaims-controller/stolostron-clusterclaims-controller-backplane-2.0.yaml:69]: same issue for repo & prefix

File: ci-operator/config/stolostron/clusterlifecycle-state-metrics/stolostron-clusterlifecycle-state-metrics-backplane-2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      name: "4.8"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 81m
+      memory: 200Mi
+tests:
```
Comment (hanqiuzh) [inline ci-operator/config/stolostron/clusterlifecycle-state-metrics/stolostron-clusterlifecycle-state-metrics-backplane-2.0.yaml:49]: no publish stage?

File: ci-operator/config/stolostron/provider-credential-controller/stolostron-provider-credential-controller-backplane-2.0.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: provider-credential-controller
+    env:
+      IMAGE_REPO: provider-credential-controller
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
```
Comment (hanqiuzh) [inline ci-operator/config/stolostron/provider-credential-controller/stolostron-provider-credential-controller-backplane-2.0.yaml:51]: same repo & prefix issue

**Pull request closed without merging**